draft-individual-openpgp-nist-bp-comp.md

```diff
@@ -276,14 +302,19 @@ informative:
         name: Victor Shoup
     date: 2003
 
+
 --- abstract
 
-This document defines PQ/T composite schemes based on ML-KEM and ML-DSA combined with ECC algorithms for the OpenPGP protocol.
-TODO
+This document defines PQ/T composite schemes based on ML-KEM and ML-DSA combined with ECC algorithms using the NIST and Brainpool domain parameters for the OpenPGP protocol.
+
 --- middle
 
 # Introduction
 
+[draft-ietf-openpgp-pqc-03] introduces post-quantum cryptography in OpenPGP.
```

Comment:
```suggestion
This document introduces additional PQ/T algorithms in OpenPGP and extends [draft-ietf-openpgp-pqc-03].
```

I just think it's better not to start the sentence / section with a reference

Reply:
I rewrote the start of the introduction accordingly.